core/chains/evm/config/toml/defaults/BOB_Mainnet.toml

```diff
@@ -0,0 +1,27 @@
+ChainID = '60808'
+# OP stack https://docs.gobob.xyz/learn/introduction/op-stack
```

Comment:
Page Not Found

Reply:
they moved the docs, right now is in https://docs.gobob.xyz/learn/introduction/stack-overview#rollup-layer.

Will update the link.